Week-4/thisKeyword/script.js

```diff
@@ -0,0 +1,42 @@
+class Person {
```

Comment:
As a suggestion, you could add a method in the Person class that could be overridden in the Student class to show polymorphism.